scraper/domains.py

```diff
@@ -254,13 +254,12 @@ def _get_product_name(self) -> str:
         return self.request_data.find("span", id="productTitle").text.strip()
 
     def _get_product_price(self) -> float:
-        return float(self.request_data.find("span", class_="a-price").span.text.replace("$", ""))
+        return float(self.request_data.find("span", class_="a-price").span.text.replace("$", "").replace(",", "").replace(" ", ""))
```

Comment:
Can please format the code using the settings in file `pyproject.toml`?

Reply:
reformatte using `black` and `ruff` where `--config` supplied `pyproject.toml`